config/canonical.json

```diff
@@ -8600,6 +8600,25 @@
       "name": "Thai Express"
     }
   },
+  "amenity/fast_food|The Habit Burger Grill": {
+    "countryCodes": ["us"],
+    "match": [
+      "amenity/fast_food|Habit Burger",
+      "amenity/fast_food|The Habit Burger",
+      "amenity/restaurant|Habit Burger",
+      "amenity/restaurant|The Habit Burger",
+      "amenity/restaurant|The Habit Burger Grill"
+    ],
+    "nocount": true,
+    "tags": {
+      "amenity": "fast_food",
+      "brand": "The Habit Burger Grill",
+      "brand:wikidata": "Q18158741",
+      "brand:wikipedia": "en:The Habit Burger Grill",
+      "cuisine": "sandwich",
```

Comment:
are you sure that it should be `"cuisine": "sandwich",` not `"cuisine": "burger",` (question based solely on name) - `cuisine=burger` is a popular tag https://taginfo.openstreetmap.org/keys/cuisine#values

Reply:
Yep this was entirely a copy/paste error on my part. I updated it to burger